styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+  max-width: 1300px;
+}
```

Comment:
[OPTIONAL] Great work applying max-width: 1300px to all your elements here 💯. However, please kindly consider adding `box-sizing: border-box;` too, it is a [CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) that will keep your project away from overflow issues, currently, your web has a small overflow in the x-axis that could be fixed using this property 🤓